.chloggen/tlscheckreceiver-implementation.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
This seems like a breaking change. Can you please also put more details in the subtext section about the configuration interface changes

Reply:
👍